src/Uno.UI/UI/Xaml/Controls/ContentPresenter/ContentPresenter.Android.cs

```diff
@@ -86,5 +91,9 @@ private void UpdateBorder(bool willUpdateMeasures)
 		{
 			UpdateBorder(true);
 		}
+
+		bool ICustomClippingElement.AllowClippingToLayoutSlot => true;
+
+		bool ICustomClippingElement.ForceClippingToLayoutSlot => CornerRadius != CornerRadius.None;
```

Comment:
Again, sounds strange that the rules are not the same for all platforms. You should at least add a comment why it's required to customize on Android in that case.

Reply:
It's because iOS is doing the clipping for rounded corners elsewhere, so the "clipping logic" must be deactivated for a control when rounded corners is activated on this platform.